cardano-cli/test/cardano-cli-golden/files/golden/help.cli

```diff
@@ -647,6 +648,21 @@ Usage: cardano-cli shelley query slot-number --socket-path SOCKET_PATH
 
   Query slot number for UTC timestamp
 
+Usage: cardano-cli shelley query ref-script-size --socket-path SOCKET_PATH
+                                                   [--cardano-mode
+                                                     [--epoch-slots SLOTS]]
+                                                   (--tx-in TX-IN)
+                                                   ( --mainnet
+                                                   | --testnet-magic NATURAL
+                                                   )
+                                                   [ --output-json
+                                                   | --output-text
+                                                   ]
+                                                   [--out-file FILE]
+
+  Calculate the reference input scripts size in bytes for provided transaction
```

Comment:
Should be transaction input. However we would also allow specification of a txids.

Reply:
Updated the text. I'll add that in the follow-up PR, is that ok?